odl/operator/operator.py

```diff
@@ -187,9 +393,10 @@ class Operator(with_metaclass(_OperatorMeta, object)):
 
     **Returns:**
 
-    `None`
+    `None` (return value is ignored)
 
-    **Out-of-place evaluation:** ``_call()``
+
+    **Out-of-place-only evaluation:** ``_call(self, x[, **kwargs])``
```

Comment:
Maybe. Another way to read this would be that the most encouraged way to implement an operator (in-place) comes first. I can change order if we want to go for "most simple first".